src/clr-addons/daterangepicker/providers/daterange-parsing.service.ts

```diff
@@ -197,26 +209,36 @@ export class DaterangeParsingService {
 
     // Check for valid date part strings.
     let year: number = parseInt(yearString);
-    if (isNaN(year)) return null;
+    if (isNaN(year)) {
+      return null;
+    }
     let month: number = parseInt(monthString);
-    if (isNaN(month)) return null;
+    if (isNaN(month)) {
+      return null;
+    }
     const date: number = parseInt(dateString);
-    if (isNaN(date)) return null;
+    if (isNaN(date)) {
+      return null;
+    }
 
     // Convert year to 4 digit year.
     year = this.parseToFourDigitYear(year);
     console.log('DaterangeParsingService.validateAndGetDate.2', {
```

Comment:
remove all console.logs at the end. this is just a reminder comment for the end of development